test/dsa_config_test_runner.sh

```diff
@@ -178,7 +182,7 @@ wq_config_test()
 		fi
 		"$ACCFG" enable-device $DSA
 		"$ACCFG" enable-wq $DSA/$WQ0
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x3 -v && echo "shoudl fail, but pass" && exit 1
+		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x3 && echo "shoudl fail, but pass" && exit 1
```

Comment:
And replace `-v` with `${VERBOSE}`, instead of just removing it?

The rest of the patch set looks fine to me.
However, the three test runners are not very concise; I'd like to clean up the code later.

Reply:
Why is there a -v there? That exact same command+options gets run 3 times, and this one is the only one with a  '-v'. I didn't a reason why it was being passed here, but none of the others. I wire it up, and all of the other DSATEST commands in dsa_config_test_runner.sh

Reply:
I believe it is for testing purposes. I noticed an open (internal)issue of IDXD related to the '-v' parameter, which indicates that dsa_test might fail with '-v'. Besides, it's not reasonable to remove it because your patch is intended to add a verbose function, so the test function should not be altered.